express/code/scripts/utils/ratings-utils.js

```diff
@@ -21,7 +21,7 @@ const RNR_API_URL = isProd ? 'https://rnr.adobe.io/v1' : 'https://rnr-stage.adob
 
 // Errors, Analytics & Logging
```

Comment:
i would update the comment to only errors & logging as i don't see analytics involved here